projects/sport-data-valley/seeds/questionnaires/daily_questionnaire.rb

```diff
@@ -146,6 +146,6 @@
   questionnaire.key = "#{File.basename(__FILE__)[0...-3]}_#{day}"
 
   questionnaire.content = { questions: current_dagboek_content, scores: [] }
-  questionnaire.title = ''
+  questionnaire.title = "#{day.capitalize} vragenlijst"
```

Comment:
- leave this title empty
- add translation_key property to questionnaires
- make sure translation_key property is exposed by questionnaire and/or response serializer
- adjust base platform questionnaire roaster so when it sets the title for an event it checks for the translation_key and if it is not nil it uses that, and otherwise the title, and otherwise the questionnaire.key.humanize (these last two conditions is how it works currently, the first one with translation key is new)
- add the translation key to the i18n files of the base platform
- adjust the frontend of the questionnaire manager view so that it also prefers showing translated translation_keys over titles when listing protocol subscriptions